Automattic/wp-calypso
# Plans 2023: Add Donations and Tips feature to grids

Related to https://github.com/Automattic/martech/issues/1744

## Proposed Changes

* Add the Donations and Tips feature
* Add tooltip text on hover `Allow your audience to support your work easily with donations and tips.`

## GIFs
### Feature grid
![2023-06-08 12 34 06](https://github.com/Automattic/wp-calypso/assets/5414230/b251be01-6be4-4ee8-aca1-91f54e622ed0)

### Comparison grid
![2023-06-08 14 57 12](https://github.com/Automattic/wp-calypso/assets/5414230/cb58256c-38f7-4091-bebf-016d432b3898)

## Testing Instructions



1. Check out this branch and run `yarn` and `yarn start` if needed. 
2. Navigate to the 2023 pricing grid in onboarding http://calypso.localhost:3000/start/plans
3. Verify that the `Donations and tips` line item is shown in the feature grid under the Free plan column. Verify that the feature tooltip is still rendered on hover
4. Verify that that the Donations and Tip line item is categorized under `Growth and Monetization Tools` in the comparison grid. Verify that the feature tooltip is rendered on hover and that there is a check mark for every column.
5. Navigate to the 2023 pricing grid in the plans page http://calypso.localhost:3000/plans/{SITE_SLUG}
6. Repeat step 3 and 4


## Pre-merge Checklist
